### ISSUE TITLE
more jbuild errors

### ISSUE THREAD

**Issue by cfcs** [mirage/ocaml-vchan#108]
on commit `1b39c3ae46f96c02deb2ad33e480b7301bf5fe4e`
```
user@pubdev:~/ocaml/ocaml-vchan$ make
jbuilder build @install --dev
Warning: Module S in lib doesn't have a corresponding .ml file.
Modules without an implementation are not recommended, see this discussion:

  https://github.com/janestreet/jbuilder/issues/9

In the meantime I'm implicitely adding this rule:

(rule (with-stdout-to s.ml
       (progn
        (echo "[@@@warning \"-a\"]\nmodule rec HACK : sig\n")
        (cat s.mli)
        (echo "\nend = HACK\ninclude HACK\n"))))

Add it to your jbuild file to remove this warning.
Error: External library "xen-evtchn-unix" not found.
-> required by "lwt/jbuild (context default)"
Hint: try: jbuilder external-lib-deps --missing --dev @install
Makefile:5: recipe for target 'build' failed
make: *** [build] Error 1
```

**Comment by cfcs** [comment]
Installing `xen-evtchn-unix` makes it build - should that be a `depends` rather than `depopts`, or am I doing something wrong again?
```
user@pubdev:~/ocaml/ocaml-vchan$ opam info vchan
             package: vchan
             version: 2.3.1
          repository: default
        upstream-url: https://github.com/mirage/ocaml-vchan/archive/v2.3.1.tar.gz
       upstream-kind: http
   upstream-checksum: 0e19dc2049706cb712f8972acf452940
            homepage: http://github.com/mirage/ocaml-vchan
         bug-reports: http://github.com/mirage/ocaml-vchan/issues
            dev-repo: http://github.com/mirage/ocaml-vchan.git
              author: Vincent Bernardoff, Jon Ludlam
             license: ISC
                tags: org:mirage
             depends: ocamlfind & lwt >= 2.5.0 & cstruct >= 1.9.0 & ppx_tools & ppx_deriving & ppx_sexp_conv & io-page & mirage-flow-lwt >= 1.0.0 & xenstore >= 1.2.2 & xenstore_transport & sexplib & cmdliner & result
             depopts: xen-evtchn-unix | xen-gnt-unix | mirage-xen
   installed-version: 
  available-versions: 0.9.5, 0.9.6, 0.9.7, 1.0.0, 2.0.0, 2.0.1, 2.0.2, 2.0.3, 2.1.0, 2.2.0, 2.3.0, 2.3.1
```

**Comment by samoht** [comment]
Fixed by #114
